avm/res/compute/gallery/main.bicep

```
@@ -140,36 +156,36 @@ module galleries_images 'image/main.bicep' = [
   for (image, index) in (images ?? []): {
     name: '${uniqueString(deployment().name, location)}-Gallery-Image-${index}'
     params: {
-      location: location
       name: image.name
+      location: image.?location ?? location
       galleryName: gallery.name
+      description: image.?description
       osType: image.osType
-      osState: image.?osState
-      publisher: image.publisher
-      offer: image.offer
-      sku: image.sku
-      minRecommendedvCPUs: image.?minRecommendedvCPUs
-      maxRecommendedvCPUs: image.?maxRecommendedvCPUs
-      minRecommendedMemory: image.?minRecommendedMemory
-      maxRecommendedMemory: image.?maxRecommendedMemory
+      osState: image.osState
+      identifier: image.identifier
+      vCPUs: image.?vCPUs ?? { min: 1, max: 4 }
+      memory: image.?memory ?? { min: 4, max: 16 }
```

```suggestion
      vCPUs: image.?vCPUs
      memory: image.?memory
```

With the default values fully implemented in the child-modules, we don't need them here anymore as they're automatically pulled if `null` is passed :)